src/components/SettingsDialog/BoardSettings/BoardSettings.tsx

```diff
@@ -0,0 +1,175 @@
+                    <SetPolicyIcon />
+                    <span>{t("BoardSettings.SetAccessPolicyPasswordProtected")}</span>
+                  </button>
+                </>
+              ) : (
+                <button className="board-settings__set-policy-button button--centered" onClick={() => handleSetPolicy("Public")}>
+                  <SetPolicyIcon />
+                  <span>{t("BoardSettings.SetAccessPolicyOpen")}</span>
+                </button>
+              ))}
+          </div>
+
+          {state.currentUserIsModerator && (
+            <>
+              <div className="settings-dialog__group">
+                <SettingsButton
+                  data-testid="author"
+                  className="board-settings__show-author-button"
+                  label={state.board.showAuthors ? t("ShowAuthorOption.hide") : t("ShowAuthorOption.show")}
+                  onClick={() => {
+                    store.dispatch(ActionFactory.editBoard({id: state.board!.id, showAuthors: !state.board.showAuthors}));
+                  }}
+                >
+                  <SettingsToggle active={state.board.showAuthors} />
+                </SettingsButton>
+                <hr className="settings-dialog__seperator" />
+                <SettingsButton
+                  data-testid="notes"
+                  className="board-settings__show-notes-button"
+                  label={state.board.showNotesOfOtherUsers ? t("ShowOtherUsersNotesOption.hide") : t("ShowOtherUsersNotesOption.show")}
+                  onClick={() => store.dispatch(ActionFactory.editBoard({id: state.board!.id, showNotesOfOtherUsers: !state.board.showNotesOfOtherUsers}))}
+                >
+                  <SettingsToggle active={state.board.showNotesOfOtherUsers} />
+                </SettingsButton>
+                <hr className="settings-dialog__seperator" />
+                <SettingsButton
+                  data-testid="columns"
+                  className="board-settings__show-columns-button"
+                  label={state.userConfiguration.showHiddenColumns ? t("ShowHiddenColumnsOption.hide") : t("ShowHiddenColumnsOption.show")}
+                  onClick={() => store.dispatch(ActionFactory.editUserConfiguration({showHiddenColumns: !state.userConfiguration.showHiddenColumns}))}
+                >
+                  <SettingsToggle active={state.userConfiguration.showHiddenColumns} />
+                </SettingsButton>
+              </div>
+
+              <SettingsButton className={classNames("board-settings__delete-button")} label={t("BoardSettings.DeleteBoard")} onClick={() => setShowConfirmationDialog(true)}>
+                <DeleteIcon />
+              </SettingsButton>
+            </>
+          )}
+
+          {showConfirmationDialog && (
+            <ConfirmationDialog
+              headline={t("ConfirmationDialog.returnToHomepage")}
```

Comment:
I think the confirmation dialog should also warn the user that the board will be deleted and cannot be recovered afterwards